### PR TITLE
project-admission-trial-period-porfolio

### PR DESCRIPTION
**Pull request by Nuri1977**
# Project from admission period (Portfolio):

1. Navbar added
2. Welcome section added
3. Work section added
4. Footer added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Nuri1977) [inline index.html:6]: Please add a responsive layout to your project. 
Free CodeCamp's responsive web design course projects are requiring implementing media queries for tablet and mobile view.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.nav {
+  background-color: #be3144;
+}
+
+#welcome-section {
+  height: 100vh;
```
Comment (Nuri1977) [inline css/style.css:6]: Please change the height of the Welcome section. Your welcome should have the dimensions as in freeCodeCamp's responsive web design course project.

File: index.html [hunk 3]
```diff
@@ -1,14 +1,145 @@
...
+
+    <section
+      id="projects"
+      class="container-fluid"
+    >
+      <div class="container text-center text-white py-4">
+        <h2 class="text-decoration-underline">These are some of my projects</h2>
+
+        <div class="row gx-4">
+          <div class="col-md-4">
```
Comment (Nuri1977) [inline index.html:76]: Please correct the height of your images in the projects sections. Please look at the image below:

portfolio_im1

In your projects section, you have used the Bootstrap grid and the height of the card containers should be the same size.

**Review by Nuri1977 — COMMENTED** [review]
### Hi @Nuri1977 

Good job so far! I know you have worked hard on making this milestone But there are some issues that you still need to work on to go to the next project but you are almost there!

unnamed


# To Highlight
- Your linters are passing 👍 
- Good commit history 🥇 
- The project runs on the machine 👍 
- Your documentation is ok 👍 

# Required changes ♻️ 

-  Kindly check the comments under the review.

# Optional suggestions 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

***Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.***

File: index.html [hunk 4]
```diff
@@ -1,14 +1,145 @@
...
+            <div class="project-tile card border-0">
```
Comment (brytebee) [inline index.html:77]: - [ ] **Required** I highly recommend adding more information to support why a change is necessary. This helps the aids in making the changes requested.
- [ ] **Optional** As much as possible, you could employ the use of emojis to express your desired changes.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi <STUDENT MENTION> @Nuri1977 👋,

Good job so far coming so far with the progress you've made! 💪💪 

However, there are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Nuri1977) [reply index.html:76]: Thank you for your instructions.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Nuri1977, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**